README.adoc

```diff
@@ -1,85 +1,90 @@
+*Step 4:* Login to the OpenShift web console using a web browser
 
-*Step 4:* Login to the OpenShift web console using a web browser (login/password - admin/admin) and select the **iot-ose** project
+**Username**: admin
+**Password**: admin
+**Project**: iot-ose
 
-image::images/OSE-projects.png[width="500", height="400", align="center"]
+image::images/OSE-projects.png[width="500", align="center"]
 
 == Configure the Zeppelin visualization tool
 
-The data visualization is stored in Notebooks. The interpreters allows Zeppelin to connect with Postgresql-backend.
+The data visualization is stored in Notebooks. The interpreters allow Zeppelin to connect to PostgreSQL.
 
 *Step 5:* Locate the *Zeppelin* service on the overview page and click on its url to launch the Zeppelin web console
 
-image::images/Zeppelin-service.png[width="500", height="400", align="center"]
-
-*Step 6:* Under notebook, select **Import note** to import the pre-configured https://github.com/ishuverma/iot-ose/blob/master/support/zeppelin/iot-ose.json[iot-ose notebook]  
-
-image::images/zeppelin-importNote.png[width="750", height="300", align="center"]
+image::images/Zeppelin-service.png[align="center"]
 
+*Step 6:* Under notebook, select **Import note** to import the pre-configured https://github.com/ishuverma/iot-ose/blob/master/support/zeppelin/iot-ose.json[iot-ose notebook]
 
-*Step 7:* Execute all visualizations by hitting the play button on the top lefthand corner of the page next to the name of the note
+image::images/zeppelin-importNote.png[align="center"]
 
-image::images/Zeppelin-results.png[width="750", height="300", align="center"]
+*Step 7:* Execute all visualizations by hitting the "Play" button on the top lefthand corner of the page next to the name of the note
 
+image::images/Zeppelin-results.png[align="center"]
 
 *Step 8:* Periodically refresh the note data through cron scheduler
 
-image::images/zeppelin-note-chron.png[width="750", height="300", align="center"]
-== Shutting Down Example Project
-The example project can be shut down either by OpenShift CLI or OpenShift web console. Follow the following order to bring down the example project:
+image::images/zeppelin-note-chron.png[align="center"]
 
-*Software Sensor -> A-MQ -> Kie -> FIS -> Zeppelin -> Postgresql*
+== Starting and Stopping the Gateway
 
-== Bringing Example Project Back Up
-The example project can be brought back up either using OpenShift CLI or OpenShift web console. Follow the following order to bring down the example project:
+The example project can be stopped using the OpenShift command-line tools or web console.  The components should be _stopped_ in the following order:
 
-*Postgresql -> A-MQ -> Kie -> FIS -> Software Sensor -> Zeppelin*
+0. Simulated Software Sensor
```

Comment:
We can probably omit the section on how to spin down the demo. Only spinning up the demo is really needed